rust/arrow/src/util/string_writer.rs

```diff
@@ -63,6 +63,7 @@
 use std::cell::RefCell;
 use std::io::{Error, ErrorKind, Result, Write};
 
+#[derive(Debug)]
```

Comment:
I think you needed this in your example to reproduce the bug, but we should not need here. May you please remove the derive here?

Reply:
removed.